src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -103,7 +110,9 @@ public DtlsTransport(Endpoint endpoint)
                 = new PacketInfoQueue(
                         getClass().getSimpleName() + "-outgoing-packet-queue",
                         TaskPools.IO_POOL,
-                        this::handleOutgoingPacket);
+                        this::handleOutgoingPacket,
+                        1024);
+        outgoingPacketQueue.setErrorHandler(queueErrorCounter);
```

Comment:
can we just create the `CountingErrorHandler` in place? `outgoingPacketQueue.setErrorHandler(new CountingErrorHandler());`

Reply:
It's static, I want one instance to count all dropped packets